### PR TITLE
Add buyerTotal and fromParticipant fields

### PR DESCRIPTION
**Pull request by ashkan18**
# Change
update with latest Exchange schema to include `buyerTotal` and `fromParticipant` to `OfferType`.

Need to wait for Exchange deploy to remove [Dont merge] but feel free to review.

### CODE REVIEW

**Comment by ashkan18** [comment]
alright this is mergeable now.

File: src/schema/ecommerce/types/offer.ts [hunk 1]
```diff
@@ -48,6 +49,10 @@ export const OfferType = new GraphQLObjectType({
         { rootValue: { userByIDLoader, partnerLoader } }
       ) => resolveOrderParty(from, userByIDLoader, partnerLoader),
     },
+    fromParticipant: {
+      type: OrderParticipantEnum,
+      description: "Offer is from which order participant",
```
Comment (williardx) [inline src/schema/ecommerce/types/offer.ts]: This wording is a little confusing -- what about something like "the order participant who created the offer"?

**Comment by williardx** [comment]
One minor comment, otherwise looks good! Feel free to merge when you're ready.

File: src/schema/ecommerce/types/offer.ts (hunk 1 above)
Comment (ashkan18) [reply src/schema/ecommerce/types/offer.ts]: makes total sesne

**Comment by ashkan18** [comment]
awesome! merging!

File: src/schema/ecommerce/types/offer.ts [hunk 2]
```diff
@@ -67,6 +72,11 @@ export const OfferType = new GraphQLObjectType({
       type: OrderInterface,
       description: "The order on which the offer was made",
     },
+    buyerTotalCents: {
+      type: GraphQLInt,
+      description: "Offer amount in cents",
+    },
+    buyerTotal: amount(({ amountCents }) => amountCents),
```
Comment (ds300) [inline src/schema/ecommerce/types/offer.ts]: didn't get a chance to review this before it was merged, but shouldn't  this be 

```
buyerTotal: amount(({ buyerTotalCents }) => buyerTotalCents),
```

?

File: src/schema/ecommerce/types/offer.ts [hunk 3]
```diff
@@ -67,6 +72,11 @@ export const OfferType = new GraphQLObjectType({
       type: OrderInterface,
       description: "The order on which the offer was made",
     },
+    buyerTotalCents: {
+      type: GraphQLInt,
+      description: "Offer amount in cents",
```
Comment (ds300) [inline src/schema/ecommerce/types/offer.ts]: Also might be good to specify in the description that buyerTotalCents includes tax and shipping.

File: src/schema/ecommerce/types/offer.ts (hunk 2 above)
Comment (ashkan18) [reply src/schema/ecommerce/types/offer.ts]: oh good catch! i'll update it now! sorry

**Pull request merged**